com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -76,6 +76,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Modified HDRP to use common FSR logic from SRP core.
 - Optimized FSR by merging the RCAS logic into the FinalPass shader.
 - Integrate a help box to inform users of the potential dependency to directional lights when baking.
+- Screen space refraction will now default to a spherical projection at ten meters instead of infinity, when no reflection probe proxy is available (1379733).
```

Comment:
Slight phrasing change: 

Screen space refraction now defaults to a spherical projection at ten meters instead of infinity when no reflection probe proxy is available (1379733).